packages/wethegit-components/src/components/accordion/accordion.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react"
+
+import styles from "./accordion.module.scss"
+import AccordionItem, { CardsData } from "./component/AccordionItem"
+import cards from "./content.js"
+
+export interface AccordionProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, "aria-label"> {
+  "aria-label": string
+  toggle: () => void
+  className?: string
+}
+
+export function Accordion({}: AccordionProps) {
```

Comment:
Doesn't look like you are using any of the props from `AccordionProps`, the object is empty